src/lib/staged_ledger/staged_ledger_diff.ml

```diff
@@ -87,6 +87,96 @@ struct
     ; creator: Compressed_public_key.t }
   [@@deriving sexp, bin_io]
 
+  module Verified = struct
+    type diff =
+      { completed_works: Transaction_snark_work.Checked.t list
+      ; user_commands: User_command.t list }
+    [@@deriving sexp, bin_io]
+
+    type diff_with_at_most_two_coinbase =
+      { diff: diff
+      ; coinbase_parts: Transaction_snark_work.Checked.t At_most_two.t }
+    [@@deriving sexp, bin_io]
+
+    type diff_with_at_most_one_coinbase =
+      { diff: diff
+      ; coinbase_added: Transaction_snark_work.Checked.t At_most_one.t }
+    [@@deriving sexp, bin_io]
+
+    type pre_diffs =
+      ( diff_with_at_most_one_coinbase
+      , diff_with_at_most_two_coinbase * diff_with_at_most_one_coinbase )
+      Either.t
+    [@@deriving sexp, bin_io]
+
+    type t =
+      { pre_diffs: pre_diffs
+      ; prev_hash: Staged_ledger_hash.t
+      ; creator: Compressed_public_key.t }
+    [@@deriving sexp, bin_io]
+  end
+
+  type verified = Verified.t [@@deriving sexp, bin_io]
```

Comment:
this is redundant and I think confusing since it's only two extra characters to refer to the `Verified.t` directly

Reply:
OK.